src/dune_console/dune

```diff
@@ -1,5 +1,5 @@
 (library
  (name dune_console)
- (libraries stdune)
+ (libraries stdune dune_notty_unix dune_config)
```

Comment:
We can't depend on `dune_notty_unix` in this library unfortunately. I suggest that you move the console size computation to stdune (or to its own library) and then access it from both dune_notty_unix and dune_console.